docs/application/dotnet/api/overview.md

```diff
@@ -1,21 +1,21 @@
 # Tizen .NET API Reference
 
-Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable(preview) applications which achieve native performance. The programming environment includes the following:
+Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieve native performance. The programming environment includes the following:
 
-- **.NET Standard API**, which implements the .NET Base Class library, allows you to use the well known C# language base class libraries and features, such as collections, threading, file I/O, and LINQ.
-- **Xamarin.Forms**, which allows you to efficiently build a user interface from standard components in C# or XAML.
-- **TizenFX API**,
-   which allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features,  such as system information  and status, battery status, sensor date, and account and connectivity services.  For the full specification, see the <a href="tizenfx/index.html" target="api">TizenFX API Reference</a>
+- **.NET Standard API** implements the .NET Base Class library, and allows you to use the well-known C# language base class libraries and features, such as collections, threading, file input/output (I/O), and Language Integrated Query (LINQ).
+- **Xamarin.Forms** allows you to efficiently build a user interface from standard components in C# or XAML.
+- **TizenFX API** allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features, such as system information and status, battery status, sensor date, and account and connectivity services.  
+  For the full specification, see the [TizenFX API Reference]( https://github.com/jkpu/tizen-docs/blob/b49532622a70fd10dec59657fe9d531cea5ab6af/docs/application/dotnet/api/tizenfx/index.html).
 
 ## .NET Standard API
 
-One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follow the .NET Standard 2.0. The column titled "2.0" in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of supported CoreFX APIs lists all the available .NET APIs.
+One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follows the .NET Standard 2.0. The column titled as 2.0 in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of the supported CoreFX APIs lists all the available .NET APIs.
```

Comment:
**The .NET APIs** is the subject, so **follow** is correct.
How can I trust your review on?